src/ai/backend/client/cli/admin/user.py

```diff
@@ -42,6 +45,7 @@ def info(ctx: CLIContext, email: str) -> None:
         user_fields["created_at"],
         user_fields["domain_name"],
         user_fields["groups"],
+        user_fields["allowed_client_ip"],
```

Comment:
I didnt add plural because @achimnol suggested that the word ips can be confusing (since ip is a abbreviation).
But after I saw the field name `groups`, ips will be okay. Anymore idea?